lib/tapjoy/ldap/cli/group/add_user.rb

```diff
@@ -0,0 +1,27 @@
+module Tapjoy
+  module LDAP
+    module CLI
+      module Group
+        # Add existing user to existing group
+        class AddUser
+          def add_user
+            puts Tapjoy::LDAP::API::Group.update(
+              opts[:group], opts[:username], :add)
+          end
+
+          private
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Keep a blank line before and after `private`.](https://www.codacy.com/app/ali-tayarani/ldap_tools/file/2634528066/issues/source?bid=3316970&fileBranchId=3318857#l12)